package.json

```
@@ -16,19 +16,19 @@
     "build:common": "npm run clean && npm run lint",
     "copy-secrets": "ncp lib/build-secrets build/build-secrets --filter=\"build-secrets($|.Dockerfile.*)\"",
     "copy-secrets:test": "ncp lib/build-secrets build/lib/build-secrets --filter=\"build-secrets($|.Dockerfile.*)\"",
-    "docgen": "typedoc --ignoreCompilerErrors --out docs --name resin-multibuild --readme README.md lib/",
+    "docgen": "typedoc --ignoreCompilerErrors --out docs --name @balena/multibuild --readme README.md lib/",
```

```suggestion
    "docgen": "typedoc --ignoreCompilerErrors --out docs --name '@balena/multibuild' --readme README.md lib/",
```
Not sure, just wondering: Is this command line interpreted in a Linux shell context where the '@' character has special meaning? What happens when you run `npm run docgen`? Does is produce further changes to `README.md`?

---

```
> @balena/multibuild@4.12.2 docgen /src
> typedoc --ignoreCompilerErrors --out docs --name @balena/multibuild --readme README.md lib/

Error: Unknown option: --ignoreCompilerErrors
Error: Unknown option: --ignoreCompilerErrors
npm ERR! code ELIFECYCLE
```

---

where is this even used?

---

I was assuming it was related to code examples in README.md because this file is mentioned on the command line. But [reading the instructions](https://www.npmjs.com/package/typedoc), I think README is actually used as an input and the `--out docs` flag refers to a `docs` folder that does not exist in the repo. In this case, my new suggestion is for this `docgen` script line to be deleted, and for `typedoc` to be removed from the `devDependencies` section.